components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentEvaluatorTest.kt

```diff
@@ -482,25 +482,25 @@ class ExperimentEvaluatorTest {
         }
 
         distribution
-                .groupingBy { it }
-                .eachCount()
-                .forEach {
-                    Assert.assertTrue(it.value in 0..25)
-                }
+            .groupingBy { it / 10 }
```

Comment:
Doesn't this mean we miss a first check here?
Maybe something like:
`.groupingBy{ it }.eachCount().forEach{ Assert.assertTrue(it.value in 0..2) }`

Reply:
Experimentally, the range that the value falls in is more like`0..9`.  I set up a loop and ran the test 1000 times to arrive at that value.  I was a little unsure as to the method used to derive the low and high bounds on the range but `0..2` seems to fail off the high bound frequently.

Reply:
I'm not sure what the exact value to me here, i just saw that it looks like we might lose some potential test coverage.